doc/api/esm.md

```diff
@@ -209,15 +217,53 @@ be interpreted as CommonJS.
 
 The `"main"` field can point to exactly one file, regardless of whether the
 package is referenced via `require` (in a CommonJS context) or `import` (in an
-ES module context). Package authors who want to publish a package to be used in
-both contexts can do so by setting `"main"` to point to the CommonJS entry point
-and informing the package’s users of the path to the ES module entry point. Such
-a package would be accessible like `require('pkg')` and `import
-'pkg/module.mjs'`. Alternatively the package `"main"` could point to the ES
-module entry point and legacy users could be informed of the CommonJS entry
-point path, e.g. `require('pkg/commonjs')`.
-
-## Package Exports
+ES module context).
+
+#### Compatibility with CommonJS-Only Versions of Node.js
+
+Prior to the introduction of Node.js’ support for ES modules, it was a common
```

Comment:
```suggestion
Prior to the introduction of support for ES modules, it was a common
```

Reply:
The point of this paragraph is to describe how the ecosystem handled ES modules before Node caught up. I think it helps clarity to explicitly say that we’re discussing before Node added support, not before “the introduction of support” which could mean when the ecosystem started supporting ES modules, or when the language added support for `import`/`export`, etc.

Reply:
In that case, the existing text is fine, but maybe consider getting rid of the possessive?

```suggestion
Prior to the introduction of Node.js support for ES modules, it was a common
``` 

or:

```suggestion
Prior to the introduction of support for ES modules in Node.js, it was a common
```

The main improvement there is typographical. Chicago Manual Style would be `Node.js's` but AP style would be (as you have) `Node.js'` and we have a mixture of both throughout the docs. Either one can look wrong/jarring to a reader used to the other style, so I"m just trying to avoid it. This is total micro-nit territory and you can absolutely ignore me. :-D

Reply:
Ah, that makes sense. `.js’` looks odd to me too, I’m happy to rephrase to avoid that. Next time please let me know that that’s what your issue was! 😄

Reply:
(One more possibility: As suggested above, just drop the apostrophe.)

```suggestion
Prior to the introduction of Node.js support for ES modules, it was a common
```